scripts/ci/tags.yaml

```diff
@@ -477,3 +477,14 @@ cellular:
     - modules/lib/cjson/
     - modules/lib/hostap/
     - modules/lib/zcbor/
+
+fast_pair:
```

Comment:
Is it possible to let this tag "select" `crypto` and `bluetooth` tags? That could simplify maintenance.
Alternatively we may consider enabling these tags manually for samples that enable `fast_pair` to simplify maintenance

Reply:
No, the approach is different. Tags are used to remove tests from scope not to added them as depending on something
(so new test/sample will be by default compiled and there needs to be added tag and dependencies to possible remove it from scope).

Similarly, it should be avoided to mix functional tags e.g. crypto bluetooth and fast_pair in single test/sample.
If only for one tag the decencies will no be fulfilled, the test will be removed from scope. Even though from different tag decencies those tests should be checked.

Reply:
Thanks for clarification

Reply:
I think we should add zephyr/subsys/settings to the list as it is common dependency of the Bluetooth subsystem and it is active in most Bluetooth samples (and its also dependency of the Fast Pair library)

Reply:
It sounds like a good idea too (since we added crypto already)